pkg/kubelet/network/plugins.go

```diff
@@ -166,6 +175,10 @@ func (plugin *noopNetworkPlugin) Name() string {
 	return DefaultPluginName
 }
 
+func (plugin *noopNetworkPlugin) Capabilities() uint {
+	return 0
```

Comment:
Souldn't this actually say that shaping is enabled to support existing behavior?